src/emulator/modules/SceGxm/src/SceGxm.cpp

```diff
@@ -605,6 +624,8 @@ EXPORT(int, sceGxmEndScene, SceGxmContext *context, const emu::SceGxmNotificatio
     glReadPixels(0, 0, width, height, GL_RGBA, GL_UNSIGNED_BYTE, pixels);
     flip_vertically(pixels, width, height, stride_in_pixels);
 
+    context->isInScene = false;
```

Comment:
Is this required? If not, can we delete it?

Reply:
There are some functions that can be called only while in scene or out scene. I added it for this purpose even if at the moment it's not used anywhere.